Model/ValidatorResult.php

```diff
@@ -0,0 +1,192 @@
+<?php
+
+namespace Mpp\UniversignBundle\Model;
+
+use Symfony\Component\OptionsResolver\Exception\AccessException;
+use Symfony\Component\OptionsResolver\Exception\InvalidOptionsException;
+use Symfony\Component\OptionsResolver\Exception\MissingOptionsException;
+use Symfony\Component\OptionsResolver\Exception\NoSuchOptionException;
+use Symfony\Component\OptionsResolver\Exception\OptionDefinitionException;
+use Symfony\Component\OptionsResolver\Exception\UndefinedOptionsException;
+use Symfony\Component\OptionsResolver\Options;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class ValidatorResult
+{
+    const STATUS_PENDING = 0;
+    const STATUS_VALID = 1;
+    const STATUS_INVALID = 2;
+
+    /**
+     * @var string
+     */
+    protected $id;
+
+    /**
+     * @var int
+     */
+    protected $status;
+
+    /**
+     * @var null|int
+     */
+    protected $reason;
+
+    /**
+     * @var null|string
+     */
+    protected $reasonMessage;
+
+    /**
+     * @var null|array
+     */
+    protected $result;
+
+    public function __construct()
+    {
+        $this->reason = null;
+        $this->reasonMessage = null;
+        $this->result = null;
+    }
+
+    /**
+     * @param OptionsResolver $resolver
+     */
+    public static function configureData(OptionsResolver $resolver)
+    {
+        $resolver
+            ->setRequired('id')->setAllowedTypes('id', ['string'])
+            ->setRequired('status')->setAllowedValues('status', [self::STATUS_PENDING, self::STATUS_INVALID, self::STATUS_VALID])
+            ->setDefault('reason', null)->setAllowedTypes('reason', ['int', 'null'])
+            ->setDefault('reasonMessage', null)->setAllowedTypes('reasonMessage', ['string', 'null'])
+            ->setDefault('result', false)->setAllowedTypes('result', ['array', 'null'])
```

Comment:
default False, but allowedTypes missing boolean ?

Reply:
Changed to null